docs/en/01-Hardware.md

```diff
@@ -7,8 +7,8 @@ You can use [Kicad 6](https://www.kicad.org/download/) to open and edit the PCB
 ![pixl.js PCB](https://github.com/solosky/pixl.js/blob/main/assets/pixljs-pcb-revc.png)
 
 # BOM
-* The BoM is summaried on the doc: [RevC Bill of Materials Buying Guide](01+1-RevC-bom)
-* The Bill of Materials in interactive format are also available, you can download it from: [RevC InteractiveHtmlBom](docs/RevC-ibom.html)
+* The BoM is summaried on the doc: [RevC Bill of Materials Buying Guide](01+1-RevC-bom.md)
+* The Bill of Materials in interactive format are also available, you can download it from: [RevC InteractiveHtmlBom]([docs/RevC-ibom.html](https://github.com/JohnVeness/pixl.js/blob/main/docs/RevC-ibom.html))
```

Comment:
I think you should link to  https://github.com/solosky/pixl.js/blob/main/docs/RevC-bom.md

Reply:
Hmm, that one isn't in English, though (in the BuyLink column).

Reply:
if this is link to RevC-ibom.html, you can link to https://github.com/solosky/pixl.js/blob/main/docs/RevC-ibom.html.
The ibom don't have i18n issue because it's a html and displayed English as default.

Reply:
Ah yes, sorry, I didn't mean to link to the one in my repo!